ArjunaJTS/jtax/tests/classes/TransactionManagerGetStatusDifferenceDemonstrator.java

```diff
@@ -94,7 +94,7 @@ public void test() throws InvalidName, SystemException,
                     // com.arjuna.orbportability.internal.orbspecific.jacorb.orb.implementations.jacorb_2_0
 					assertTrue(
 							"Status: " + getStatusSync.getTransactionManagerGetStatus(),
-							getStatusSync.getTransactionManagerGetStatus() == Status.STATUS_NO_TRANSACTION);
```

Comment:
@mmusgrov please can you take a look at this? When consolidating other parts of the build system to use JacORB 3.7 (something partially done in https://github.com/jbosstm/narayana/pull/1364/) it shows this has changed and now for JacORB when a Jakarta Transactions `Synchronization` is invoked in this manner it gets `Status.STATUS_COMMITTED` rather than `Status.STATUS_NO_TRANSACTION`. JacORB is then now consistent with the other ORBs and local JTA mode.

Reply:
In regards to using 3.7 of JacORB, do you know if that version no longer needed whatever the the special patched version mentioned here had?: https://github.com/jbosstm/narayana/blob/master/ArjunaCore/scripts/jts-jacorb-setup-env.sh#L48

Reply:
I don't think the patched version is necessarily specific to narayana rather they were general fixes maintained by jboss/nick. I've looked at some of the patches in [1] and they seem like general fixes. I don't know whether or not Nick pushed the fixes to the jacorb project. That said, we only seem to use the bundled jacorb.jar in [2,3] and I don't know what those services are for (they are either historical or are used for the JTS + Corba Name Service in a docker container work that Gytis did [4] but, without checking it is just a guess.

Regarding the question about the Synchronization returning Status.STATUS_COMMITTED rather than Status.STATUS_NO_TRANSACTION: my input here is that since the transaction context is undefined in the after synch (according to the spec) the actual return value can be either. I would assume that we changed it because returning the actual status is more informative and was requested.

But in summary, since we are about to move JacORB and since the only issue seems to be what the after synch returns, I think we can explain this difference and move on, WDYT?

[1] https://repository.jboss.org/jacorb/
[2] ./ArjunaCore/arjuna/services/config/recoveryservice.conf
[3] ./ArjunaJTS/jts/services/config/transactionservice.conf
[4] https://github.com/jboss-dockerfiles/narayana

Reply:
@jmfinelli This comment provides more context about why would like to quickly follow up with your PR.

Reply:
The bundled JacORB is what is used by standalone distributions: https://github.com/jbosstm/narayana/blob/master/narayana-full/src/main/assembly/bin.xml and during `qa/` tests: https://github.com/jbosstm/narayana/blob/master/qa/TaskImpl.properties.template#L40 (though this PR changes both of these to use the versions defined in poms now)